src/routes/demographics/Content.svelte

```diff
@@ -85,13 +85,13 @@
         { key: "decline", label: "Decline to identify" },
       ],
     },
-    hispanicLatinoSpanishOrigin: {
-      key: "hispanicLatinoSpanishOrigin",
-      label: "3. Are you of Hispanic, Latino, or Spanish origin?",
+    hispanicLatinxSpanishOrigin: {
+      key: "hispanicLatinxSpanishOrigin",
```

Comment:
Since we're changing these, [this](https://github.com/mozilla-rally/rally-core-addon/blob/a0a46c4e2d8a157dae665ad3ba4b4fe344ed4912/core-addon/DataCollection.js#L191) must be changed as well. Please make sure other occurrences are changed (if present) :)